src/plugins/analysis/hash/code/hash.py

```diff
@@ -34,11 +34,11 @@ def process_object(self, file_object):
         If you want to propagate results to parent objects store a list of strings 'summary' entry of your result dict
         '''
         file_object.processed_analysis[self.NAME] = {}
-        for h in self.hashes_to_create:
-            if h in algorithms_available:
-                file_object.processed_analysis[self.NAME][h] = get_hash(h, file_object.binary)
+        for hash_ in self.hashes_to_create:
+            if hash_ in algorithms_available:
+                file_object.processed_analysis[self.NAME][hash_] = get_hash(hash_, file_object.binary)
             else:
-                logging.debug('algorithm {} not available'.format(h))
+                logging.debug('algorithm {} not available'.format(hash_))
```

Comment:
tzz, now here was an obvious refactoring-to-fstring-possibility

Reply:
changed